server/integration_tests/test_data/multisv/query_2/chart_config.json

```diff
@@ -476,39 +476,39 @@
             "statVar": "Count_Person_1OrMoreYears_PovertyStatusDetermined_DifferentHouseAbroad"
           },
           "Count_Person_5OrMoreYears_LanguageOtherThanEnglishSpokenAtHome_BelowPovertyLevelInThePast12Months_multiple_place_bar_block": {
-            "name": "Population: 5 Years or More, Language Other Than English, Below Poverty Level in The Past 12 Months",
+            "name": "5 Years or More, Language Other Than English, Below Poverty Level in Past 12 Months",
             "statVar": "Count_Person_5OrMoreYears_LanguageOtherThanEnglishSpokenAtHome_BelowPovertyLevelInThePast12Months"
           },
           "Count_Person_5OrMoreYears_LanguageOtherThanEnglishSpokenAtHome_BelowPovertyLevelInThePast12Months_multiple_place_bar_block_pc": {
             "denom": "Count_Person",
-            "name": "Population: 5 Years or More, Language Other Than English, Below Poverty Level in The Past 12 Months",
+            "name": "5 Years or More, Language Other Than English, Below Poverty Level in Past 12 Months",
             "statVar": "Count_Person_5OrMoreYears_LanguageOtherThanEnglishSpokenAtHome_BelowPovertyLevelInThePast12Months"
           },
           "Count_Person_5OrMoreYears_OnlyEnglishSpokenAtHome_BelowPovertyLevelInThePast12Months_multiple_place_bar_block": {
-            "name": "Population: 5 Years or More, Only English, Below Poverty Level in The Past 12 Months",
+            "name": "5 Years or More, Only English, Below Poverty Level in Past 12 Months",
```

Comment:
Feels like Population is the important piece here... would add back in without the colon, ditto the related svs

Reply:
Done